controllers/vmware/test/controllers_suite_test.go

```diff
@@ -65,7 +65,10 @@ func getTestEnv() (*envtest.Environment, *rest.Config) {
 	utilruntime.Must(vmwarev1.AddToScheme(scheme.Scheme))
 
 	// Get the root of the current file to use in CRD paths.
-	_, filename, _, _ := goruntime.Caller(0) //nolint
+	_, filename, _, ok := goruntime.Caller(0)
+	if !ok {
+		klog.Fatalf("Failed to get information for current file from runtime")
+	}
```

Comment:
```suggestion
	Expect(ok).To(BeTrue(), "Failed to get information for current file from runtime")
```
Small improvement. This way we trigger a regular gomega error instead of os.Exit vs klog.Fatal (same in similar cases of course)